resources/views/external-link-confirm.blade.php

```diff
@@ -11,10 +11,16 @@
     x-data="{
         openModal() {
             Livewire.emit('openModal', '{{ $identifier }}')
+        },
+        redirect() {
+            window.open('{{ $url }}', '_blank')
+        },
+        hasDismissedModal() {
```

Comment:
not really `dismissed` 🤔 would benefit from a better name to avoid confusion later

Reply:
I'm dismissing this.